pkg/appgateserver/sdkadapter/relay_client.go

```diff
@@ -38,21 +38,18 @@ func NewRelayClient(
 func (r sdkRelayClient) SendRequest(
 	ctx context.Context,
 	urlStr string,
-	body []byte,
-	method string,
-	header map[string][]string,
+	requestBz []byte,
 ) ([]byte, error) {
 	requestUrl, err := url.Parse(urlStr)
 	if err != nil {
 		return nil, err
 	}
 
-	bodyReader := io.NopCloser(bytes.NewReader(body))
+	bodyReader := io.NopCloser(bytes.NewReader(requestBz))
 	defer bodyReader.Close()
 
 	request := &http.Request{
-		Method: method,
-		Header: header,
+		Method: http.MethodPost,
```

Comment:
Should we leave a TODO to figure out what we want to do with the headers in the future?

Reply:
The whole request (header, body, method and url) is serialized in `requestBz` now, no no need to care about it at this level.

The current `Method: http.MethodPost` is just how the `AppGateServer` and `RelayMiner` are communicating, which is independent from which method and headers the `RelayMiner` will use to query the `BackendService` (these are extracted back from `requestBz`)

Reply:
Makes sense. #PUC